website/content/en/docs/getting-started/eksctl.yaml

```diff
@@ -4,7 +4,8 @@ kind: ClusterConfig
 metadata:
   name: ${CLUSTER_NAME}
   region: ${AWS_DEFAULT_REGION}
-nodeGroups:
+  version: "1.20"
```

Comment:
I was going to set this version, but I realized that it will cause our doc to go stale quickly. It's a bit tricky -- perhaps we could add a note/step upgrade eksctl on mac/linux to ensure the latest version?

Reply:
Unless you think we should just maintain this?

Reply:
The docs will need to be versioned at some point so we're going to have to maintain versions either way. Some companies will have really old versions of EKS and we'll want to keep karpenter docs for them and maybe in the future we need to stop karpenter users from using the latest EKS (for whatever reason).